app/models/hackbot/conversations/check_in.rb

```diff
@@ -143,6 +151,10 @@ def wait_for_notes(event)
 
       private
 
+      def record_notes(event)
+        data['notes'] = event[:text] unless event[:text] =~ /^(no|nope|nah)$/i
```

Comment:
I honestly don't think you need a separate method for this. I'd just create a method for checking to see if a user says no and then use that.

```
def state_method
  data['notes'] = event[:text] unless no?(event[:text])
end

private

def no?(text)
  text =~ /^(no|nope|nah)$/i
end
```